### ISSUE TITLE
IntelliSense does not work correctly with -include compiler option

### ISSUE THREAD

**Issue by b1r1b1r1** [microsoft/vscode-cpptools#5675]
**Type: LanguageService**

**Describe the bug**
- OS and Version: Mac OS Mojave 10.14.4
- VS Code Version: 1.46.1
- C/C++ Extension Version: 0.28.3
- Other extensions you installed (and if the issue persists after disabling them): no
- Does this issue involve using SSH remote to run the extension on a remote machine?: no
- Explanation: 
When using -include compiler option, intellisense fails to see #define's

**Steps to reproduce**

1) Create 2 files in a single workspace:

a.h
```c++
#define A
```

a.cpp
```c++
#include <stdio.h>

#ifdef A
int a = 1;
#endif

int main() {
    printf("%d\n", a);
}
```
1) Add 
```
"compilerArgs": [
    "-includea.h"
]
```
option to c_cpp_properties.json
2) Open a.cpp
3) Look at color of #ifdef block

**Expected behavior**

A should be defined and #ifdef block must be enabled.
Instead this block is disabled

**Screenshots**

![problem](https://i.imgur.com/06z23cc.png)

The following code does compile with `Apple clang version 11.0.0 (clang-1100.0.33.17)` or GNU g++.

**Comment by Colengms** [comment]
Closing this as a dupe of: https://github.com/microsoft/vscode-cpptools/issues/5512
This appears to already have been fixed, and is pending release in 0.29.

**Comment by sean-mcmanus** [comment]
Should be fixed with 0.29.0.
